src/ol/format/wmtscapabilitiesformat.js

```diff
@@ -353,6 +367,23 @@ ol.format.WMTSCapabilities.TMS_LINKS_PARSERS_ = ol.xml.makeParsersNS(
  * @type {Object.<string, Object.<string, ol.xml.Parser>>}
  * @private
  */
+ol.format.WMTSCapabilities.DIMENSION_PARSERS_ = ol.xml.makeParsersNS(
+    ol.format.WMTSCapabilities.NAMESPACE_URIS_, {
+      'Default': ol.xml.makeObjectPropertySetter(
+          ol.format.XSD.readString),
+      'Value': ol.xml.makeObjectPropertyPusher(
+          ol.format.XSD.readString)
+    }, ol.xml.makeParsersNS(ol.format.WMTSCapabilities.OWS_NAMESPACE_URIS_, {
+      'Identifier': ol.xml.makeObjectPropertySetter(
```

Comment:
```ows:Identifier``` is not specific to WMTS. OWS contains common things for multiple OGC specs, see: https://github.com/openlayers/ol3/blob/master/src/ol/format/owsformat.js and how this is used in the WMTS parser already